src/config.js

```diff
@@ -8,8 +11,8 @@ export default {
   useDevTools: process.env.DEV_TOOLS || false,
   print: {
     labelConfig: {
-      codeFormat: { 4: 2 },
-      canSelectPath: { 4: true, 5: true }
+      codeFormat: { '4': 2 },
```

Comment:
Have you done is as a result of getting flow type errors? Or is this an actual improvement?